04-store-api/starter/routes/products.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const router = express.Router();
+
+const {
+  getAllProducts,
+  getAllProductsStatic,
+} = require("../controllers/products");
+
+router.route("/").get(getAllProducts);
+router.route("/static");
+router.get("/static", getAllProductsStatic); // Separately. For some reason, if I do it together, there's an error
```

Comment:
When you tried separately, did you setup as `router.route("/static").get(getAllProductsStatic)`?